files/default/handler/audit_report.rb

```diff
@@ -143,12 +143,13 @@ def load_automate_fetcher
         require 'chef-automate/fetcher'
       end
 
-      def get_opts(format, quiet, attributes)
+      def get_opts(reporter, quiet, attributes)
         output = quiet ? ::File::NULL : $stdout
-        Chef::Log.debug "Format is #{format}"
+        Chef::Log.debug "Reporter is [#{reporter}]"
         opts = {
           'report' => true,
-          'format' => format,
+          'format' => reporter,      # For compatibility with older versions of inspec. This line can be removed from Q2 2019
```

Comment:
On the InSpec side of these we try to phrase these as `TODO:` so we can search for it easily.

Could we do that here or open an issue? I won't hold up the PR over it.